src/lv_core/lv_obj_style_dec.h

```diff
@@ -199,6 +199,99 @@ _LV_OBJ_STYLE_SET_GET_DECLARE(SCALE_END_COLOR, scale_end_color, lv_color_t, _col
 
 #undef _LV_OBJ_STYLE_SET_GET_DECLARE
 
+
+static inline void lv_obj_set_style_local_pad_all(lv_obj_t * obj, uint8_t part, lv_state_t state, lv_style_int_t value)
+{
+    _lv_obj_set_style_local_int(obj, part, LV_STYLE_PAD_TOP | (state << 8), value);
```

Comment:
Perhaps it would be better to call the wrappers (e.g. `lv_obj_set_style_local_pad_top`), as that centralizes the logic for handling the state and style enums in one place. It's also an inline function so there shouldn't be any runtime cost for doing so.

Reply:
Good, I'll change it. I was thinking of the overhead, but let's trust the compiler :)